usaspending_api/data_load/fpds_loader.py

```diff
@@ -0,0 +1,314 @@
+from os import environ
+import psycopg2
+from collections import OrderedDict
+import logging
+import time
+import math
+
+from usaspending_api.data_load.field_mappings_fpds import (
+    transaction_fpds_columns,
+    transaction_normalized_columns,
+    transaction_normalized_functions,
+    legal_entity_columns,
+    legal_entity_boolean_columns,
+    legal_entity_functions,
+    recipient_location_columns,
+    recipient_location_functions,
+    place_of_performance_columns,
+    place_of_performance_functions,
+    award_functions,
+    transaction_fpds_boolean_columns,
+    transaction_fpds_functions,
+)
+from usaspending_api.data_load.data_load_helpers import (
+    subtier_agency_list,
+    capitalize_if_string,
+    false_if_null,
+    format_value_for_sql,
+)
+
+# DEFINE THESE ENVIRONMENT VARIABLES BEFORE RUNNING!
+USASPENDING_CONNECTION_STRING = environ["DATABASE_URL"]
+BROKER_CONNECTION_STRING = environ["DATA_BROKER_DATABASE_URL"]
+
+CHUNK_SIZE = 1000
+
+logger = logging.getLogger("console")
+
+
+class Timer:
```

Comment:
Is Timer not centrally defined for importing?

Reply:
iirc no. I know I had a chat with @sethstoudenmier about this, and there was some reason that management commands can't use the normal timer, so @kbard always included a home-spun one.

Reply:
Management commands can use the project `Timer` class/function.  It's the standalone scripts that can't.

Well, they could, but that would require us to pull it out of Django and make it a standalone library... maybe a future enhancement.